Extensions/Today/TodayViewController.swift

```diff
@@ -25,6 +25,9 @@ private struct TodayUX {
     static let copyLinkImageWidth: CGFloat = 23
     static let margin: CGFloat = 8
     static let buttonsHorizontalMarginPercentage: CGFloat = 0.1
+    static let privateSearchButtonColorThree = UIColor(red: 117.0/255.0, green: 41.0/255.0, blue: 167.0/255.0, alpha: 1.0)
```

Comment:
Instead of one two or three could we give them some better name. 
Eg. privateSearchButtonColourLightGreen